src/Settings.php

```diff
@@ -654,6 +666,18 @@ public function admin_page_display( string $plugin_page ): void {
 		);
 	}
 
+	/**
+	 * Show notice to inform users that the Donate button settings are now in Appearance > Menus.
+	 *
+	 * @return string
+	 */
+	private function show_donate_menu_notice(): string {
+		return '<p>These settings have been moved to <a href="/wp-admin/nav-menus.php?action=locations">a new menu location</a>
```

Comment:
I mentioned it in the previous PR, so I'll say it here again 😁 I came up with this copy when I first worked on this, it should probably be reviewed by @comzeradd or Suzi!

[Our documentation](https://planet4.greenpeace.org/manage/admin-setup/#donate-button) for this section will also likely need an update, and we should add somewhere some directions for how to add a Donate button dropdown menu since it's a bit complicated.

Reply:
I agree! I left this suggestion into the description of the [base PR](https://github.com/greenpeace/planet4-master-theme/pull/1628) just to follow up before merge into master.